### PR TITLE
multiple ternary operator challenge - add note about code readability

### LINKED ISSUE #36260: Multiple ternary [why are you teaching this]?
https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/basic-javascript/use-multiple-conditional-ternary-operators

It's important that young programmers understand how this works., but why isn't there any emphasis not to use multiple ternary? It's just bad programming.

### PR DESCRIPTION
**Pull request by kozlovvski**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #36260 
---

Hey, as @ojeytonwilliams suggested in #36260, I've updated [Basic JavaScript: Use Multiple Conditional (Ternary) Operators](https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/basic-javascript/use-multiple-conditional-ternary-operators) to include note about proper code readability when using multiple ternary operators.

However, I've done that only for English - I don't know any of the other languages.

What do you think about it?

### CODE REVIEW

**Review by thecodingaviator — APPROVED** [review]
~LGTM~

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/use-multiple-conditional-ternary-operators.english.md [hunk 1]
```diff
@@ -28,6 +28,16 @@ The above function can be re-written using multiple <code>conditional operators<
 
 ```js
 function findGreaterOrEqual(a, b) {
+  return (a === b) ? "a and b are equal" 
+    : (a > b) ? "a is greater" 
+    : "b is greater";
+}
+```
+
+However, this should be used with care - using multiple <code>conditional operators</code> without proper indentation may make your code hard to read:
```
Comment (thecodingaviator) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/use-multiple-conditional-ternary-operators.english.md]:
```suggestion
However, this should be used with care as using multiple <code>conditional operators</code> without proper indentation may make your code hard to read. For example:
```

**Review by thecodingaviator — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/use-multiple-conditional-ternary-operators.english.md [hunk 2]
```diff
@@ -28,6 +28,16 @@ The above function can be re-written using multiple <code>conditional operators<
...
+  return (a === b) ? "a and b are equal" 
+    : (a > b) ? "a is greater" 
+    : "b is greater";
+}
+```
+
+However, this should be used with care as using multiple <code>conditional operators</code> without proper indentation may make your code hard to read. For example:
+
+```js
+function (a, b) {
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/use-multiple-conditional-ternary-operators.english.md]:
```suggestion
function findGreaterOrEqual(a, b) {
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Please add the missing function name.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/use-multiple-conditional-ternary-operators.english.md (hunk 2 above)
Comment (kozlovvski) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/use-multiple-conditional-ternary-operators.english.md]: Wow, how could I miss that 😮

**Review by ojeytonwilliams — APPROVED** [review]

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/use-multiple-conditional-ternary-operators.english.md (hunk 2 above)
Comment (ojeytonwilliams) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/use-multiple-conditional-ternary-operators.english.md]: Easy mistake to make, don't worry about it!

**Comment by ojeytonwilliams** [comment]
@thecodingaviator I'm happy with this now, feel free to merge if you are.

**Review by thecodingaviator — APPROVED** [review]
LGTM

**Comment by thecodingaviator** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**